functions/src/custom/vanity.ts

```diff
@@ -104,6 +104,86 @@ const create_link = async (vanity: Vanity): Promise<void> => {
     });
 };
 
+export const build_vanity_link_v2 = async (request_data: {
```

Comment:
What's the difference between `create_vanity_link` and `build_vanity_link_v2`?

Reply:
```suggestion
export const build_vanity_link_v2 = async ({
```
I don't see the need to wrap everything inside `request_data` since the very first line of the function is to destructure the arguments back again.... 

Can remove this line from the function `const { first_name, last_name, email, destination, primary_domain, subdomain, slashtag } = request_data;`